app/src/main/java/com/tatar/currencify/ui/util/UIExtensions.kt

```diff
@@ -0,0 +1,113 @@
+package com.tatar.currencify.ui.util
+
+import com.tatar.currencify.R
+
+fun String.toCurrencyString(): String {
+    return when (this) {
+        CURRENCY_CODE_AUD -> "Australian Dollar"
```

Comment:
Woow, that's something. I think my enum suggestion would work on this kind of case, what you think?

Reply:
instead of creating every code const string.

Reply:
Enums can be used instead of the String but this time I have to create enum type  for each currency. Does not really add a value.